homeassistant/components/mysensors/__init__.py

```diff
@@ -34,7 +44,7 @@
 
 DEFAULT_BAUD_RATE = 115200
 DEFAULT_TCP_PORT = 5003
-DEFAULT_VERSION = "1.4"
+DEFAULT_VERSION = "2.3"
```

Comment:
I am not sure either, but don't you think it is unlikely that a new deployment would be using an old version?
Maybe we should not have a default at all and force the user to enter the version. An empty free-text field might be confusing for the user though. Maybe we should have no default value and 2.3 as suggested value?

I fully agree auto-detect would be amazing.

Reply:
I'm thinking that 1.4 may be the safest version to default to since it won't activate features that require a higher version.

I'll like the suggestion with requiring input version and have a suggested value. :+1: